app/src/main/java/net/gsantner/markor/format/csv/README.md

```diff
@@ -0,0 +1,32 @@
+.../app/src/main/java/net/gsantner/markor/format/csv/README.md
```

Comment:
Please don't put non-code files into java code directory. If you want to make code comments/code documentatoin, leave it in code (javadoc), for markdown files, theres the `<gitrepo>/doc/` folder.

Reply:
moved to .../doc/CSV_README.md